zero/dg_iz_cu.cu

```diff
@@ -15,16 +15,17 @@ extern "C" {
 #include <gkyl_dg_iz_priv.h>
 #include <gkyl_util.h>
 #include <gkyl_const.h>
+#include <gkyl_array_ops_priv.h>
 }
 
 __global__ static void
 gkyl_iz_react_rate_cu_ker(const struct gkyl_dg_iz *up, const struct gkyl_range conf_rng, const struct gkyl_range adas_rng,
   const struct gkyl_basis *adas_basis, const struct gkyl_dg_prim_vars_type *calc_prim_vars_elc_vtSq,
   const struct gkyl_dg_prim_vars_type *calc_prim_vars_donor_gk, const struct gkyl_array* moms_elc,
-  const struct gkyl_array* moms_donor, struct gkyl_array* vtSq_elc, struct gkyl_array* prim_vars_donor,
+  const struct gkyl_array* moms_donor, struct gkyl_array* vtSq_elc, struct gkyl_array* vtSq_iz, struct gkyl_array* prim_vars_donor,
   struct gkyl_array* coef_iz, struct gkyl_array* ioniz_data, int num_basis, enum gkyl_react_self_type type_self,
   double mass_elc, double elem_charge, double E, double maxLogTe, double minLogTe, double dlogTe,
-  double maxLogM0, double minLogM0, double dlogM0, int resTe, int resM0)
+  double maxLogM0, double minLogM0, double dlogM0, int resTe, int resM0, long nc)
```

Comment:
@tnbernard best not to pass nc, you can just get it from `vtSq_iz` inside the kernel (that way your CPU and GPU code will look more similar)